src/tests/packages.config

```diff
@@ -1,10 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="CR.CodeStyle.CSharp.Full" version="1.0.1" targetFramework="net471" developmentDependency="true" />
```

Comment:
Code Styles unnecessary in test projects (although no need to remove them now they've been added.).